INSTALL.md

```diff
@@ -7,92 +7,123 @@ which includes this INSTALL.md file, is denoted $(CERTIFIER).
 The Ubuntu 20.04 install guide can be found in
 [Ubuntu Install](./Doc/install-certifier-Ubuntu-20.04.md).
 
-Read the Guide in the Doc directory to understand the basic nomenclature
-and programs you will build and use.
+Read the [Guide](Doc/Guide.pdf) in the Doc directory to understand the
+basic nomenclature and programs you will build and use.
 
 
-Certifier API
--------------
+# Certifier API
 
-The certifier API is in the src directory.  To compile and run the
-certifier API tests:
+The certifier API is in the src/ directory.
 
-  cd $(CERTIFIER)/src
-  make clean -f certifier_tests.mak
-  make -f certifier_tests.mak
-  ./certifier_tests.exe [--print_all=true]
+## Setup steps
 
+You will need to install the following packages:
+
+```shell
+$ sudo apt-get update -y
+$ sudo apt-get install -y protobuf-compiler
+$ sudo apt-get install -y libgtest-dev/jammy
+$ sudo apt-get install -y libgflags-dev/jammy
+$ sudo apt-get install -y protoc-gen-go
+$ sudo apt-get install -y golang-go
+$ sudo apt-get install -y libmbedtls-dev/jammy
+```
+
+## To compile and run the certifier API tests
+
+```shell
+  $ cd $(CERTIFIER)/src
+  $ make -f certifier_tests.mak clean
+  $ make -f certifier_tests.mak
+  $ ./certifier_tests.exe [--print_all=true]
+```
```

Comment:
Instructions are the same as prior, just re-formatted in `shell` format, for improved readability.